lib/html_parser.dart

```diff
@@ -247,7 +257,7 @@ class HtmlParser extends StatelessWidget {
     RenderContext newContext = RenderContext(
       buildContext: context.buildContext,
       parser: this,
-      style: context.style.copyOnlyInherited(tree.style),
+      style: context.style?.copyOnlyInherited(tree.style) ?? Style.fromTextStyle(Theme.of(context.buildContext).textTheme.bodyText2),
```

Comment:
Why is this needed now?

Reply:
Its not, this was fixed by a different PR of mine. It can be removed before merging, this was just temporary to avoid a "red screen of death" upon launching the app.